snomed/com.b2international.snowowl.snomed.api.rest.tests/src/com/b2international/snowowl/snomed/api/rest/components/SnomedRelationshipApiTest.java

```diff
@@ -60,6 +64,8 @@
  */
 public class SnomedRelationshipApiTest extends AbstractSnomedApiTest {
 
+	private static final String ROOT_ISA_RELATIONSHIP_ID = "1019504021";
```

Comment:
The prefix `ROOT_` could be changed to `RELEASED_` or similar across all test cases; it is a bit misleading at the moment, and it doesn't really matter who the "owner" is, as long as the component is released.